pkg/virtual/apiexport/controllers/apireconciler/apiexport_apireconciler_reconcile.go

```diff
@@ -279,6 +283,7 @@ type apiResourceSchemaApiDefinition struct {
 
 	UID          types.UID
 	IdentityHash string
+	ClaimLabels  string
```

Comment:
what is this?

Reply:
This is left over from a previous attempt. Need to remove.

Reply:
Actually, I'm not sure what this is as it was introduced in a [commit](https://github.com/kcp-dev/kcp/pull/2456/commits/a0f06b7fc8abb46f3ef2cde34a0e6db3524351eb) I got from @s-urbaniak. I believe he was working with Andy on admission at the time.

Removing it causes the server to fail to start. Serg, would you have any insight?

Reply:
We either have to do this, or have a way to modify the label selector for existing storage when claim labels change. I think modifying would be better but will require more work.